portfolio/src/main/webapp/script.js

```diff
@@ -115,28 +116,90 @@ function createCommentElement(comment) {
+function deleteTask(commentId) {
   const params = new URLSearchParams();
-  params.append('id', comment.id);
+  params.append('id', commentId);
   fetch('/delete-comment', {method: 'POST', body: params});
+  window.location.replace("index.html");
 }
 
-/* When the user clicks on the button, 
-toggle between hiding and showing the dropdown content */
+/**Onclicking the commentSettigns button show the dropdown-content */
 function commentSettings() {
-  document.getElementById("myDropdown").classList.toggle("show");
+  document.getElementById("dropdown-content").style.visibility = "visible"; 
+  document.getElementById("dropdown-content").style.display = "inline-block"; 
+}  
+
+/**Reload along with posting the maxComments to url, when user selects the no. of comments he wants to see. */
+function sendForm() {
+  window.location.replace("index.html?maxComments="+ document.getElementById('quantity').value );
+}
+
+/**Set the default value to maxComments when user doesn't give any input or when the page first loads. */
+function getMaxComments() {
+    const defaultValue = 15;
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    if(urlParams.get('maxComments') == null)
+        return defaultValue;
+    else
+        return(urlParams.get('maxComments'));
 }
 
-// Close the dropdown if the user clicks outside of it
-window.onclick = function(event) {
-  if (!event.target.matches('.commentSettings')) {
-    var dropdowns = document.getElementsByClassName("dropdown-content");
-    var i;
-    for (i = 0; i < dropdowns.length; i++) {
-      var openDropdown = dropdowns[i];
-      if (openDropdown.classList.contains('show')) {
-        openDropdown.classList.remove('show');
-      }
+/**Makes the form visible and adds the 'action' to it by fetching 
+the url(that the form needs to post to) from the servlet */
+function fetchBlobstoreUrlAndShowForm() {
+  fetch('/blobstore-upload-url')
+      .then((response) => response.text())
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('upload-images');
+        messageForm.action = imageUploadUrl;
+        messageForm.classList.remove('hidden');
+      });
+}
+
+/**Fetching all image urls stored in datastore and presenting them on screen along with date/time and its message */
+function getAndShowImages(){
+  const imageListElement = document.getElementById('images-container');
+
+  fetch("/my-image-servlet").then((response) => response.json()).then((imageDetails) => {
+    if(imageDetails.length==0) {
+        imageListElement.innerText = "No images";
+    }
+    else {
+        imageDetails.forEach((imageDetail) => {
+            imageListElement.appendChild(createImageElement(imageDetail));
+        });
     }
-  }
+  })
+}
+
+/**Creating an image element */
+function createImageElement(imageDetail) {
+  const imageElement = document.createElement('li');
+  imageElement.className = 'image';
```

Comment:
You are mixing single- and double-quoted strings in this function. You should pick one and be consistent.

You should only switch between the single- and double-quoted strings in order to avoid escaping when using quotes inside the strings.

Reply:
Oops sorry, will make sure that it doesn't repeat.